jobs/subseasonal/stats/JEVS_SUBSEASONAL_STATS

```diff
@@ -1 +1,111 @@
-sample
+#!/bin/bash
+
+set -xa
+
+# ###################################
+# SET SHELL PROCESSING VARIABLES
+# ###################################
+export PS4='$SECONDS + ' 
+date
+
+export machine="WCOSS2"
+
+###########################################################
+# obtain unique process id (pid) and make temp directories
+###########################################################
+export pid=$$
+#export DATA_IN=${DATA_IN:-$DATAROOT}
+export DATA_IN=${DATA_IN:-$DATAROOTtmp}
+export DATA=$DATA_IN/${pid}
+
+
+#rm -rf $DATA_IN
+mkdir -p $DATA
+cd $DATA 
+
+###############################
+# Determine Job Output Name on System
+###############################
+export pgmout="OUTPUT.${pid}"
+
+
+
+################################################################
+# SENDCOM=YES--Copy files from TMPDIR to $COMOUT
+# SENDECF=YES--Flag events on ecflow
+# SENDDBN=YES--Issue DBNet Client Calls
+################################################################
+export SENDCOM=${SENDCOM:-NO}
+export SENDDBN=${SENDDBN:-NO}       # need to set to NO for testing
+export SENDECF=${SENDECF:-NO}
+export SENDDBN_NTC=${SENDDBN_NTC:-NO}
+################################################################
+# Specify Execution Areas
+################################################################
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/$envir/dcom}
+export HOMEevs=${HOMEevs:-${PACKAGEROOT}/evs.${evs_ver}}
+export EXECevs=${EXECevs:-$HOMEevs/exec}
+export PARMevs=${PARMevs:-$HOMEevs/parm}
+export USHevs=${USHevs:-$HOMEevs/ush}
+export FIXevs=${FIXevs:-$HOMEevs/fix}
+
+if [ $MODELNAME = gefs ] ; then
+    export members="30"
+fi
+if [ $MODELNAME = cfs ] ; then
+    export members="4"
+fi
+
+###################################
+# Set up the UTILITIES
+###################################
+# for wwgrib: find the path of wgrib2: 
+#export WGRIBpath=`echo $WGRIB | sed 's/\/wgrib//'`
+
+# Run setpdy and initialize PDY variables
+export cycle=${cycle:-t${cyc}z}
+setpdy.sh
+. ./PDY
+
+#export VDATE=$PDYm6
+
+export VDATE=$(date -d "today -22 day" +"%Y%m%d")
+
```

Comment:
The VDATE variable should be defined such that it can inherit a value previously defined in the upstream ecf script.
Change line 70 to:
`export VDATE=${VDATE:-$PDYm2}`
Remove line 72. The override definition will be set in the ecf script.